filebeat/input/log/harvester.go

```diff
@@ -360,7 +360,16 @@ func (h *Harvester) onMessage(
 		jsonFields = f.(common.MapStr)
 	}
 
+	// Add meta.created in all messages when `add_event_metadata` is enabled
+	// from Elastic Common Schema Reference:
+	// https://www.elastic.co/guide/en/ecs/current/ecs-event.html#_event_field_details
 	var meta common.MapStr
+	if h.config.AddEventMetadata {
+		meta = common.MapStr{
+			"created": message.Ts,
+		}
+	}
```

Comment:
The `message.TS` here is already the parsed docker container read timestamp. We would need to introduce a 'ReadTS' to `message` in order to track when Filebeat did start reading the message.